app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -17,32 +17,84 @@ keywords:
 
-No matter what settings you train and work in – whether it's mainstream schools, special schools, or both – you’ll work with pupils with complex needs.
+### Becoming a special educational needs co-ordinator
 
-$q-jane-wilkinson$
+A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of pupils with special educational needs and disabilities in mainstream schools. SENCOs also support teachers and teaching assistants in a range of ways including providing training, observing lessons and developing learning plans for pupils. 
 
-You can [find a teacher training course](https://find-teacher-training-courses.service.gov.uk/) and filter by 'courses with a SEND specialism' if you have a particular interest in the area.
+As a SENCO you will get to know the specific needs of pupils who need extra support, so you can help them to feel comfortable and confident before teaching starts. 
 
-However, it’s not essential to do a course specialising in SEND.
+To become a SENCO, you’ll need to gain [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) through initial teacher training and then complete a mandatory [SENCO national professional qualification (SENCO NPQ)](https://www.gov.uk/guidance/special-educational-needs-co-ordinators-national-professional-qualification). You do not need to complete the NPQ before taking a SENCO role, you have 3 years to complete the qualification. 
 
-As you train and teach you'll get the skills you need to work with disabled pupils and pupils with special educational needs.
+As well as focusing on supporting pupils with SEND, many SENCOs will also teach in their main subject area. 
 
-This will enable you to work in a range of settings throughout your career.
+### Mandatory qualification in sensory impairment
 
-## Teach pupils with sensory impairments
+To teach pupils with hearing impairments, vision impairments or multi-sensory impairments you’ll need a [mandatory qualification in sensory impairment (MQSI)](https://www.gov.uk/guidance/mandatory-qualifications-specialist-teachers) in addition to QTS.  
 
-You need [specific qualifications to teach a class of pupils with hearing impairments, vision impairments or multi-sensory impairments](https://www.gov.uk/guidance/mandatory-qualifications-specialist-teachers).
+An MQSI will give you the skills needed to recognise pupils’ barriers to learning and help them overcome these. 
 
-## Become a special educational needs coordinator
+## Training for teaching pupils with SEND
 
-A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of disabled pupils and pupils with special educational needs.
+All initial teacher training courses cover teaching pupils with SEND. Once you have gained QTS, you can look for courses that specialise in SEND to develop your training further. 
```

Comment:
I'm a bit confused by this. I read it as you have to get QTS first, before you specialise in SEND. But the courses on Find seem to let you get QTS and specialise in SEND on one course?

Reply:
Good point, and not sure, will check with policy!